conf/modules/call_snv_deepvariant.config

```diff
@@ -24,40 +24,31 @@ process {
     withName: '.*CALL_SNV_DEEPVARIANT:DEEPVARIANT' {
         ext.args = { "--model_type=${params.analysis_type.toUpperCase()}" }
         ext.prefix = { "${meta.id}_deepvar" }
-        publishDir = [
-            path: { "${params.outdir}/deepvariant" },
-            mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-        ]
     }
 
     withName: '.*CALL_SNV_DEEPVARIANT:GLNEXUS' {
         ext.args = '--config DeepVariant_unfiltered'
-        publishDir = [
-            enabled: false
-        ]
     }
 
     withName: '.*CALL_SNV_DEEPVARIANT:SPLIT_MULTIALLELICS_GL' {
         ext.args = '--output-type z --multiallelics -both'
-        publishDir = [
-            enabled: false
-        ]
     }
 
     withName: '.*CALL_SNV_DEEPVARIANT:REMOVE_DUPLICATES_GL' {
         ext.args = '--output-type z --rm-dup none'
         ext.prefix = { "${meta.id}_split_rmdup" }
         publishDir = [
-            path: { "${params.outdir}/glnexus" },
+            enabled: !params.dedicated_mt_analysis,
```

Comment:
Don't we want to save this anyways since otherwise we will only save an annotated vcf?

Reply:
Not quite.. When we run the dedicated mitochondrial analysis workflow, we will get the contents of this file with the mitochondrial variants filtered out from the process, RAREDISEASE:GATK4_SELECTVARIANTS.

Reply:
👍